test/ModuleInterface/option-preservation.swift

```diff
@@ -0,0 +1,11 @@
+// REQUIRES: OS=macosx
+// RUN: %target-swift-frontend -enable-resilience -target arm64-apple-ios8.0 -emit-interface-path %t.swiftinterface -module-name t -emit-module -o /dev/null %s
```

Comment:
Yes, this is intentional: I wanted to limit the test to when I'm on-and-targeting macosx, and then explicitly pass a _new_ target flag such that I can check that it got deposited in the .swiftinterface and then picked up and deposited in the resulting .swiftmodule, all without accidentally getting confused by a false positive because (say) we just happen to be building on or for a target that's the same as the explicit flag I'm testing.

Reply:
Okay, but you can't always build for iOS just because you're running on macOS. Someone might have turned that platform off. You'd probably be okay with `-parse-stdlib`, I guess.